hw19-dz26/js/hw-26-paparazzi.js

```diff
@@ -0,0 +1,14 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+   function structureUserInfo(userName) {
+       let result = {};
+       result.userName = userName;
+       return function(role) {
+           result.role = role;
+           return result;
+       }
+   }
```

Comment:
not bad, but see how you can optimize a function to return a function that returns an object with passed arguments
```javasript
function structureUserInfo(userName) {
  return function (role) {
    return { userName: userName, role: role };
  };
}

console.log(structureUserInfo("John")("Admin"));
```
Note that this entry can also be shortened ```{ userName: userName, role: role }``` like ```{userName, role}```
https://www.geeksforgeeks.org/shorthand-syntax-for-object-property-value-in-es6/